Automattic/woocommerce-payments
# Prevent WooPay OTP after click Place Order button

Fixes #4792

#### Changes proposed in this Pull Request









* Stop WooPay OTP request after the user clicks the `Place Order`.

#### Testing instructions





* On the merchant store, log in with a user that's registered in WooPay (you can also finish an order on WooPay and then try to order again, this way the e-mail field will be pre-filled).
* Add a product to the cart and proceed to check out.
* Click on "Place Order" before the email field receives focus.
* The SMS OTP modal should not be shown as the request to the WooPay `/exists` page was aborted.

-------------------


**Post merge**
